1_5_3 and 1_5_4/includes/templates/YOUR_TEMPLATE/templates/tpl_modules_main_product_image.php

```diff
@@ -0,0 +1,24 @@
+<?php
```

Comment:
I based this solely on the fact that the Zen Cart core files are different. I know I use the version info in the header when comparing. If I was to not see 1.5.5 in this I'd be concerned it was not merged or updated correctly requiring me go back and compare previous versions. I don't see a downside to including versions that match up with core file and folder structure.